scripts/novaseq/stage/demux-novaseq.bash

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+
+set -eu
+
+ulimit -n 4096
+
+##########
+# PARAMS #
+##########
+
+IN_DIR=${1?'please provide a run dir'}
+DEMUXES_DIR=${2?'please provide the demuxes dir'}
+FC=${3?'fc_id needed'}
+PROJECTLOG=${4?'projectlog needed'}
+
+RUN=$(basename ${IN_DIR})
+OUT_DIR=${DEMUXES_DIR}/${RUN}
+SCRIPT_DIR=/home/proj/${ENVIRONMENT}/bin/git/demultiplexing/scripts/novaseq/
+#EMAIL=clinical-demux@scilifelab.se
```

Comment:
Is this right? 🙉

Reply:
Again, this is a script ready for development and testing, it saves people from wrecking prod ready scripts, which can be risky.